appinventor/appengine/war/tutorials/Tutorial_TalkToMe.js

```diff
@@ -0,0 +1,135 @@
+/* Step Object */
+// step = function(text, validate, url){
+// 	this.text=text;
+// 	this.validate=validate;
+// 	this.url=url;
+// }
+
+
+/* Tutorial Object */
+
+/*Every tutorial will be harcoded into a tutorial object which can easily be run */
+
+var Tutorial_TalkToMe = {
+	title: "Talk To Me",
+	difficulty: "easy",
+	noCheck: function(){
+		return true;
+	},
+	steps:
+		[
+			{
+				text: "Welcome to App Inventor! To get started, we'll be guiding you through the process of building an app that allows your phone to talk to you with the press of a button.",
```

Comment:
I'm not sure why it looks so funny here. In Intellij, it looks like the indentation is consistently at 2 spaces.